frontend/components/NumberInput.tsx

```diff
@@ -0,0 +1,39 @@
+import { InputNumber, InputNumberProps } from 'antd';
+
+export const allowOnlyNumbers = (e: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
I'd rather do non-exported, if you think we'll need to reuse it somewhere, better to place in `utils/formValidations.ts` (similar to mono repo)

Reply:
no worries